### PR TITLE
create random integer helper function action

### PR DESCRIPTION
**Pull request by zalmanlew**
**Random Integer**
Generates a random integer between two specified values.
Useful for using in a future step (such as implementing a random delay).

### CODE REVIEW

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

File: components/helper_functions/actions/random-integer/random-integer.mjs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    max: {
+      type: "integer",
+      default: 10000,
+      label: "Max",
+      description: "Highest possible integer",
+    },
+  },
+  run() {
+    return {
```
Comment (vellames-turing) [inline components/helper_functions/actions/random-integer/random-integer.mjs:25]: I would say to return it directly
```suggestion
    return Math.floor(Math.random() * (this.max - this.min + 1) + this.min)
```

**Review by vellames-turing — APPROVED** [review]
Tks!

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Helper_Function_2937.pdf](https://github.com/PipedreamHQ/pipedream/files/8786993/Helper_Function_2937.pdf)

**Pull request merged**